CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ v1.7.4 [unreleased]
 ### Bugfixes
 - [#315](https://github.com/latera/homs/pull/315) Add additional exceptions for Oracle's date formats.
 - [#316](https://github.com/latera/homs/pull/316) Fix the way ajax request are made in lookup select.
+- [#323](https://github.com/latera/homs/pull/323) Submit each form on it's own button.
```

Comment:
It looks like we lost sync between master and 1.7 changelogs. Could you sync it please?